test/UnitBadDocLoad.cpp

```diff
@@ -73,7 +73,7 @@ UnitBase::TestResult UnitBadDocLoad::testBadDocLoadFail()
 
         // Extract all json entries into a map.
         items = JsonUtil::jsonToMap(dialog.substr(sizeof("jsdialog:")));
-        auto firstId = items["id"];
+        const std::string firstId = items["id"];
```

Comment:
Also a good idea in general, README.CONTRIBUTING.md recommends against auto in this case.